### PR TITLE
Feature: Add accessibility identifiers to the tabbar elements

### PR DESCRIPTION
**Pull request by asboxi**
# Description
To make it easy to automate the UI, this PR includes accessibility identifiers to the tab bar.

### CODE REVIEW

File: Example/Controllers/SwiftUI/MiniAppDashboardView.swift [hunk 1]
```diff
@@ -23,6 +23,7 @@ struct MiniAppDashboardView: View {
                 }
                 .tabItem {
                     Label("List I", systemImage: "list.bullet")
+                        .accessibilityIdentifier(AccessibilityIdentifiers.tabBarListI)
```
Comment (laubengaier) [inline Example/Controllers/SwiftUI/MiniAppDashboardView.swift:26]: @asboxi congratz to raising your first PR 🎉 
one thing to mention is you need to add the identifier at the end of each enum value eg to get the proper full format

```suggestion
.accessibilityIdentifier(AccessibilityIdentifiers.tabBarListI.identifier)
```

Comment (asboxi) [reply Example/Controllers/SwiftUI/MiniAppDashboardView.swift]: Thanks for the comment. Fixed!

**Review by laubengaier — APPROVED** [review]
lgtm, I'll let CI run

**Review by Climbatize — APPROVED** [review]
Awesome!

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:
🥳 .Welcome to the dev team 😉

**Pull request merged**